src/components/ProfilePicture.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, BoxProps, Skeleton } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { fetchProfilePicture } from '../api/cristinApi';
+
+interface ProfilePictureProps {
+  id: string;
+  height: string;
+  hasBorder?: boolean;
```

Comment:
Disse kan like godt sendes via `sx` props også, eller?
```suggestion
```

Reply:
Om vi skal ha identisk border flere plasser kan det gi mening å ha en egen `hasBorder` prop da

Reply:
hmm, ser nå at `height` brukes for både img og Skeleton, så da gir det mer mening.. kanskje bare sette en default value for height da? Antar det er en størrelse som er langt mer vanlig enn andre..?